src/components/Global/Header/Header.module.scss

```diff
@@ -0,0 +1,76 @@
+$white: white;
+$black: black;
+$primary-color: #F16B67;
+
+.header {
+    width: 100vw;
+    height: 7vh;
+    background-color: $white;
+    padding: 0 2vw;
+
```

Comment:
header should also have a small grey border at the bottom. `border-bottom: rgba($grey, 0.05) 1px solid;` is what I use